Pipfile

```diff
@@ -44,6 +44,7 @@ aiomcache = "==0.6.*"
 # aiocache will automatically use ujson when available. ujson supports decimal types, and json does not.
 ujson = "*"
 httptools = "*"
+aio-snowplow-tracker = "*"
```

Comment:
Great job on addressing the asyncio issue. I see that you forked the repo under your personal account. Maybe we should move this fork to pocket or Mozilla github org so that anyone can update the package with the upstream repo if needed?

Reply:
Let's see if it becomes a blocker. I used my personal account because I did this work in my spare time. Over the last year, Snowplow has made 3 changes to their Python library, none of which Pocket really needs: https://github.com/snowplow/snowplow-python-tracker/commits/master.